src/chrome/content/rules/ebay.com.sg.xml

```diff
@@ -0,0 +1,195 @@
+<!--
+
+	For other eBay coverage, see ebay.com.xml	
```

Comment:
Need also to be added there

Reply:
Added in https://github.com/EFForg/https-everywhere/pull/14626/commits/009a46792aa559d2146894613c875cff3bc78fd0